spanner/tests/system.py

```diff
@@ -531,6 +536,60 @@ def test_read_w_manual_consume(self):
         self.assertEqual(streamed._current_row, [])
         self.assertEqual(streamed._pending_chunk, None)
 
+    def test_read_w_index(self):
+        ROW_COUNT = 2000
```

Comment:
- At the implementation level, Python doesn't have *any* [constants](https://en.wikipedia.org/wiki/Constant_(computer_programming)):  any name can be rebound at any time.
- This is a symbolic name for a literal value, which is not intended to be changed during its lifetime, which is the semantic of "constantness".
- The notion that constants can only be declared at module scope is not backed by PEP8, which says, "Constants are usually defined on a module level "  That is not a prescription that they may not be defined at other scopes.